src/main/java/com/thealgorithms/geometry/GrahamScan.java

```diff
@@ -61,14 +55,14 @@ public GrahamScan(Point[] points) {
     }
 
     /**
-     * @return A stack of points representing the convex hull.
+     * @return An iterable collection of points representing the convex hull.
      */
     public Iterable<Point> hull() {
-        Stack<Point> s = new Stack<>();
+        Stack<Point> result = new Stack<>();
```

Comment:
Is it important use here Stack? If we replace it via LinkedList for example, than we will have only:
`return new LinkedList<>(hull);`

Reply:
rollbacked 👍